### ISSUE TITLE
AOD

### ISSUE THREAD

**Issue by ithustle** [kaltura/nginx-vod-module#970]
Is this lib works with Audio On demand? If yes, please, can you provide an example? I want to build a streaming server for audio on demand where my files are stored on AWS S3.

**Comment by erankor** [comment]
Yes, audio is supported, all the sample configurations will work for audio-only as well

**Comment by ithustle** [comment]
And what about my files stored on AWS S3? Can you provide an example?

**Comment by erankor** [comment]
There are some threads about that as well... e.g. #717

**Comment by ithustle** [comment]
Thanks. I'll check it out.

**Comment by ithustle** [comment]
> Yes, audio is supported, all the sample configurations will work for audio-only as well

Hello again, @erankor .
I'm working with mp3 files and hls protocol and none of my files plays
